docs/cli.md

```diff
@@ -368,6 +368,220 @@ Output will look like this, where `python` will be green as it is the package th
+│   │   ├── ruamel.yaml.clib v0.2.8
+│   │   │   ├── python_abi v3.12 (*)
+│   │   │   └── python v3.12.2 (*)
+│   │   └── python v3.12.2 (*)
+│   ├── jsonschema v3.2.0
+│   │   ├── python v3.12.2 (*)
+│   │   ├── pyrsistent v0.20.0
+│   │   │   ├── python_abi v3.12 (*)
+│   │   │   └── python v3.12.2 (*)
+│   │   ├── six v1.16.0
+│   │   └── attrs v23.2.0
+│   │       └── python v3.12.2 (*)
+│   ├── requests v2.31.0
+│   │   ├── idna v3.6
+│   │   │   └── python v3.12.2 (*)
+│   │   ├── python v3.12.2 (*)
+│   │   ├── urllib3 v2.2.1
+│   │   │   ├── pysocks v1.7.1
+│   │   │   │   └── python v3.12.2 (*)
+│   │   │   ├── python v3.12.2 (*)
+│   │   │   └── brotli-python v1.1.0
+│   │   │       ├── python v3.12.2 (*)
+│   │   │       ├── python_abi v3.12 (*)
+│   │   │       └── libcxx v16.0.6 (*)
+│   │   ├── certifi v2024.2.2
+│   │   │   └── python v3.12.2 (*)
+│   │   └── charset-normalizer v3.3.2
+│   │       └── python v3.12.2 (*)
+│   ├── click v8.1.7
+│   │   └── python v3.12.2 (*)
+│   ├── pykwalify v1.8.0
+│   │   ├── python v3.12.2 (*)
+│   │   ├── docopt v0.6.2
+│   │   ├── python-dateutil v2.9.0
+│   │   │   ├── python v3.12.2 (*)
+│   │   │   └── six v1.16.0 (*)
+│   │   └── ruamel.yaml v0.18.6 (*)
+│   └── python v3.12.2 (*)
+└── tbump v6.9.0
+    ├── cli-ui v0.17.2
+    │   ├── colorama v0.4.6
+    │   │   └── python v3.12.2 (*)
+    │   ├── python v3.12.2 (*)
+    │   ├── tabulate v0.9.0
+    │   │   └── python v3.12.2 (*)
+    │   └── unidecode v1.3.8
+    │       └── python v3.12.2 (*)
+    ├── python v3.12.2 (*)
+    ├── schema v0.7.5
+    │   ├── contextlib2 v21.6.0
+    │   │   └── python v3.12.2 (*)
+    │   └── python v3.12.2 (*)
+    ├── tomlkit v0.12.4
+    │   └── python v3.12.2 (*)
+    └── docopt v0.6.2 (*)
+```
+
+A regex pattern can be specified to filter the tree to just those that show a specific direct dependency:
+
+```shell
+➜ pixi tree pre-commit
+└── pre-commit v3.3.3
+    ├── virtualenv v20.25.1
+    │   ├── filelock v3.13.1
+    │   │   └── python v3.12.2
+    │   │       ├── libexpat v2.6.2
+    │   │       ├── readline v8.2
+    │   │       │   └── ncurses v6.4.20240210
+    │   │       ├── libsqlite v3.45.2
+    │   │       │   └── libzlib v1.2.13
+    │   │       ├── bzip2 v1.0.8
+    │   │       ├── libzlib v1.2.13 (*)
+    │   │       ├── libffi v3.4.2
+    │   │       ├── tk v8.6.13
+    │   │       │   └── libzlib v1.2.13 (*)
+    │   │       ├── xz v5.2.6
+    │   │       ├── ncurses v6.4.20240210 (*)
+    │   │       └── openssl v3.2.1
+    │   ├── platformdirs v4.2.0
+    │   │   └── python v3.12.2 (*)
+    │   ├── distlib v0.3.8
+    │   │   └── python v3.12.2 (*)
+    │   └── python v3.12.2 (*)
+    ├── pyyaml v6.0.1
+    │   ├── python_abi v3.12
+    │   ├── python v3.12.2 (*)
+    │   └── yaml v0.2.5
+    ├── nodeenv v1.8.0
+    │   └── python v3.12.2 (*)
+    ├── python v3.12.2 (*)
+    ├── cfgv v3.3.1
+    │   └── python v3.12.2 (*)
+    └── identify v2.5.35
+        └── python v3.12.2 (*)
```

Comment:
Again, would shrink this so maybe 10 or so. Using `...` as a common insert to show there has been a cut off in the text.

Reply:
Snipped, but I went a little longer to show marking and tree depth.

```shell
➜ pixi tree
├── pre-commit v3.3.3
│   ├── cfgv v3.3.1
│   │   └── python v3.12.2
│   │       ├── libsqlite v3.45.2
│   │       │   └── libzlib v1.2.13
│   │       ├── libzlib v1.2.13 (*)
│   │       ├── libffi v3.4.2
│   │       ├── libexpat v2.6.2
│   │       ├── readline v8.2
│   │       │   └── ncurses v6.4.20240210
│   │       ├── xz v5.2.6
│   │       ├── openssl v3.2.1
│   │       ├── bzip2 v1.0.8
│   │       ├── tk v8.6.13
│   │       │   └── libzlib v1.2.13 (*)
│   │       └── ncurses v6.4.20240210 (*)
│   ├── pyyaml v6.0.1
...
├── rust v1.76.0
│   └── rust-std-aarch64-apple-darwin v1.76.0
├── openssl v3.2.1
├── pkg-config v0.29.2
│   ├── libglib v2.78.4
│   │   ├── libiconv v1.17
│   │   ├── gettext v0.21.1
│   │   │   └── libiconv v1.17 (*)
...
```